src/api/mod.rs

```diff
@@ -1,30 +1,60 @@
 use std::env;
 
+use actix_cors::Cors;
+use actix_web::{
+    dev::Server,
+    web::{self, Data},
+    App, HttpServer,
+};
 use dotenv::dotenv;
-use rocket::routes;
 
-use crate::db;
+use crate::db::{self, Database, Pool};
 
-pub mod cves;
-pub mod products;
+mod cves;
+mod error;
+mod products;
+mod utils;
 
-#[get("/")]
-pub fn index() -> &'static str {
-    "pong"
-}
-
-pub fn run() {
+pub fn run() -> Result<Server, anyhow::Error> {
     dotenv().ok();
 
+    let host = "0.0.0.0";
```

Comment:
Wouldn't mind these as default but ideally we do not hardocde the host/port. It seems that the current Rocket implementation allows us to pass [`ROCKET_ADDRESS/PORT`](https://github.com/Exein-io/kepler/blob/daa0cb100fa3be71ab4d2c112f38c044943d240b/docker-compose.yml#L28-L29). Could we refactor those to something more generic and default to `0.0.0.0:8000` if missing?

Reply:
very basic implementation. a better configuration method should be implemented but not in this pr, opening a new issue